app/controllers/static_pages_controller.rb

```diff
@@ -13,4 +15,10 @@ def terms_of_use; end
   def success_stories
     @success_stories = SuccessStory.all
   end
+
+  def import_content
+    github_paths = Lesson.pluck(:github_path)
+    UpdateLessonContentJob.perform_async(github_paths)
```

Comment:
I see what you mean about it being a pain with fetching twice.

I think it would be worth creating a new ImportAllLessonContentJob` job and just calling the[ import_all](https://github.com/TheOdinProject/theodinproject/blob/main/app/services/lesson_content_importer.rb#L13) method on the import service. The functionality is different enough to justify it imo.

Edit:
Actually instead of calling it `ImportAllLessonContentJob` can we create a lessons folder within the jobs directly and namespace the lesson import and update jobs please mate?
```
lessons
  - update_content_job.rb
  - import_all_content_job.rb
```

Reply:
Sounds good :+1: